### PR TITLE
[redis] Address redis 3.0.0 release

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
`redis-py==3.0.0` [has been released](https://github.com/andymccurdy/redis-py/releases/tag/3.0.0) which includes changes that break our integration.

This has brought attention to an unpinned dependency in our `ddtrace-run` tests which has been addressed. The listed version support for the integration has also been updated.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -148,7 +148,7 @@ deps =
     celery40: celery>=4.0,<4.1
     celery41: celery>=4.1,<4.2
     celery42: celery>=4.2,<4.3
-    ddtracerun: redis
+    ddtracerun: redis==2.10.6
```
Comment (brettlangdon) [inline tox.ini]: can probably just do `redis>=2.6,<3.0` here, we do want to make sure we are compatible with `2.10.7` if it comes out, or `2.11.0`

Comment (Kyle-Verhoog) [reply tox.ini]: I'd say that the version of redis shouldn't matter here. We use redis as a tool to test `ddtrace-run` and it doesn't make sense for the test to be dependent on any version of redis (so long as it works).

Comment (brettlangdon) [reply tox.ini]: sure, makes sense

**Comment by brettlangdon** [comment]
This has been resolved with #537 and will be released with `0.17.0` early next week.

**Pull request closed without merging**